src/campaign/sass/components/_feedbackBanner.scss

```diff
@@ -0,0 +1,53 @@
+.feedback-banner {
+    margin: 0 -80px;
+    padding: 0 80px;
+    background-color: black;
+    color: #D2D2D2;
+    display: block;
+}
+
+.feedback-banner--hidden{
+    display: none;
+}
+.feedback-banner__content {
+    margin: 0px 0;
+    color: #D2D2D2;
+
+    font-size: 16px;
+    @include media-query($from: tablet) {
+        font-size: 19px;
+    }
+}
+
+.feedback-banner__link {
+    margin: 0px 0;
+    color: #D2D2D2;
+    text-decoration: none;
+    border-bottom: 1px solid #fff;
+
+    @include media-query($from: tablet) {
+        font-size: 19px;
+    }
+
+    &:hover, &:focus {
+        border-bottom-color: transparent;
+    }
+}
+
+.feedback-banner__content--close {
+    float: right;
+    width:20%;
+}
+
+.feedback-banner__content--close-button {
+    float: right;
+}
+.feedback-banner__content--text {
+    float: left;
+    width: 80%;
+}
+.feedback-banner--inner {
+    display: inline-block;
+    width: 100%;
+    padding: 5px 0 5px 0;
```

Comment:
This could be shortened to `padding: 5px 0` but I would suggest increasing this a little bit to match the design maybe: `padding: 10px 0`